### ISSUE TITLE
Telemetry error on 2021 Russia Qualifying

### ISSUE THREAD

**Issue by hash55** [theOehrly/Fast-F1#72]
Sorry I'm having an error when I try to get the telemetry of any car of the russian gp quali.
For instance, if I do `laps.pick_driver('NOR').pick_fastest().get_telemetry()` the program crashes.  
KeyError: SessionTime
![Captura de pantalla 2021-12-19 223702](https://user-images.githubusercontent.com/84033043/146691701-daca73b4-5bc0-4784-a412-d0530ffb8ad5.png)

**Comment by asw-v4** [comment]
Hello,

I have tried to replicate your issue. 

Firstly I tried this:
```
race = ff1.get_session(2021, 'Russian Grand Prix', 'Q')
laps = race.load_laps()
print(laps.pick_driver('NOR').pick_fastest().get_telemetry())
```
And this caused some errors that were not explicitly clear as to why. A key error isnt all that helpful in the grand scheme of things.

I did also notice that load_laps has a `with_telemetry` flag. So i passed this value as true and it seems to work.
```
race = ff1.get_session(2021, 'Russian Grand Prix', 'Q')
laps = race.load_laps(with_telemetry=True)
print(laps.pick_driver('NOR').pick_fastest().get_telemetry())
```

Hope this helps :)

**Comment by theOehrly** [comment]
@hash55 can you report back if what @asw-v4 describes fixes your problem?
When I test this, I do get a different error if I do not pass ``load_telemetry=True``. Therefore, I suppose that your issue is actually a different one?
Are you using the latest version of FastF1? (v2.1.12)

@asw-v4 true, the KeyError is not very helpful. This should be improved. Just for context, telemetry loading is separated and disabled by default, as F1 did have some server problems concerning specifically telemetry data in the past.

**Comment by hash55** [comment]
@asw-v4 Thanks for your help but I had already the telemetry loaded. (with_telemetry=True)
@theOehrly Indeed, I am using the last version of fastf1, and I can load all drivers of all races except for russia quali. In the same script, if I change the quali session to race, it works perfectly
However, I have just realized that during that quali session, I tried to "record" the live timing manually, using
 `python -m fastf1.livetiming save saved_data.txt` . So what I did is to delete that folder from cache, re-run the script, and it's solved!
It was a silly mistake by me, thanks for the fast replies
